source/content/guides/drupal-9/drupal-9-hosted/05-create-site.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Migrate a Drupal 8 Site to Drupal 9
+subtitle: Create the Drupal 9 Site
+description: 
+cms: "Drupal 9"
+categories: [develop]
+tags: [code, launch, migrate, site, updates]
+contributors: [wordsmither]
+reviewed: "2021-03-31"
+layout: guide
+permalink: docs/guides/drupal-9-hosted/create-site
+anchorid: create-site
+editpath: hosted/05-create-site.md
+---
+
+1. Create a new Drupal 9 Integrated Composer site from the Upstream using the: [Create Your Pantheon Site](https://dashboard.pantheon.io/sites/create?upstream_id=897fdf15-992e-4fa1-beab-89e2b5027e03) page. On the this page:
+
+   - Enter a name for the site.
+
+   - Choose an Organization if applicable.
+
+   - Choose a region for the site. Click **Continue**.
+
+   After you click Continue, it will take several minutes for the platform to set everything up.
+
+1. In the **<span class="glyphicons glyphicons-wrench"></span> Dev** tab on the Dashboard, set the site's Development Mode to [SFTP](/sftp#sftp-mode).
+
+1. Still in the Dev tab, click **Visit Development Site** and follow the prompts to complete the CMS installation.
+
+1. Return to the Dev tab, set the site's Development Mode to Git, and [clone the site locally](/local-development#get-the-code).
+
+1. In your local terminal, from the project root directory, run `composer install`.
+
+<Accordion title="If your existing site has modules incompatible with MariaDB 10.4" id="consider-confirm-mariadb">
+
+<Partial file="drupal-9/drupal-9-mariadb-considerations.md" />
+
+<Partial file="confirm-db-upgrade-workflow.md" />
+
+</Accordion>
```

Comment:
In its current state, users will start from a D8 site and without any D9 upgrade preparation will end with a D9 site. This could potentially break their site. I'd recommend adding something like https://pantheon.io/docs/guides/composer-convert#set-drupal-core-version after this create site step